### PR TITLE
Make the Twig integration compatible with Twig 2.0

### PR DESCRIPTION
**Pull request by stof**

### CODE REVIEW

File: src/Assetic/Extension/Twig/AsseticFilterFunction.php [hunk 1]
```diff
@@ -11,19 +11,14 @@
 
 namespace Assetic\Extension\Twig;
 
-class AsseticFilterFunction extends \Twig_Function
+class AsseticFilterFunction extends \Twig_SimpleFunction
```
Comment (stof) [inline src/Assetic/Extension/Twig/AsseticFilterFunction.php]: Note that in Assetic 2.0, we would be able to stop using a custom class entirely. but I need to keep it like that for now for BC reasons.
I have some ideas in mind to refactor the Twig integration layer in Assetic to make the bundle cleaner (no overwrite of everything to handle the Symfony integration) but I will keep it for a separate PR.
Preparing the code for removal of this custom class will also be part of that delayed work.

**Comment by Soullivaneuh** [comment]
:+1:

@stof Could this be merged and tagged? Thanks.

**Comment by stof** [comment]
@Soullivaneuh this is planned during the weekend (either this evening or tomorrow)

**Comment by stof** [comment]
@Soullivaneuh the release will happen tommorow

**Comment by Soullivaneuh** [comment]
Thanks. :+1:

**Pull request merged**